### DISCUSSION TITLE
[Feature] Some suggested additional features:

### DISCUSSION THREAD

**Discussion by pyccl** [immich-app/immich#12212] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

1.Homepage Categorization: On the homepage, clearly differentiate between photos, videos, and live photos on the left-hand side. Rename the existing "Photos" section to "Medias" (to encompass all types of media, with videos and photos displayed on the same page). Introduce new sections: "Photos" (solely for photos, excluding videos), "Videos" (exclusively for videos, without photos), and "Live Photos" (strictly for live photos, separate from videos and static photos).
2.Selective Upload Location: Allow users to choose the location for uploading photos and videos, such as a specific folder within the "External Gallery". Additionally, add an "Upload" button within the "Folders" page, so that uploaded videos and photos are automatically saved into the selected folder.
3.User-Bound "People" Section: Integrate "People" with user accounts. When a user logs in, they can directly access all photos and videos associated with the "people" they have bound, without the need for sharing permissions.
4.Enhanced Photo and Video Management in "Tags" and "Folders": Extend the functionality of the "Tags" and "Folders" pages to include photo and video management options, such as deletion and moving (shifting a photo or video to another folder). This will enable users to organize and manage their content more efficiently.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
As requested in the template, please stick with only one feature per post.

**Reply by pyccl** [reply]
oh.thanks,
